config/model.js

```diff
@@ -0,0 +1,28 @@
+const db = require("../database/dbConfig");
+
```

Comment:
great job making a helper model file, it will be easy to expand this apps functionality